ui/src/dashboards/components/TemplateControlBar.js

```diff
@@ -74,4 +112,4 @@ TemplateControlBar.propTypes = {
   isOpen: bool,
 }
 
-export default TemplateControlBar
+export default ControlBarWrapper
```

Comment:
This export is not being used anywhere, and in fact `TemplateControlBar` is still being imported in `Dashboard.js`. Did something get accidentally left out of this PR?